README.md

```diff
@@ -46,3 +46,13 @@ This project requires specific versions of the following tools. To make sure you
    ```sh
    yarn install
    ```
+
+## Terraform Configurations
+
+All infrastructure-related configurations are within the `infra` folder, except for the `.terraform-version` file necessarily defined at the root level to set the Terraform version for the entire repository.
+
+In details:
+
+- `identity`: user-assigned managed identity definition used by the GitHub workflows of this repository. Must be run from local environment and high-level roles are required on target subscription
```

Comment:
what do you mean with 'local environment', the developer laptop?

Reply:
Yes I do